tests/misra/test_misra.sh

```diff
@@ -53,8 +53,8 @@ printf "\n${GREEN}** PANDA H7 CODE **${NC}\n"
 cppcheck $PANDA_OPTS -DSTM32H7 -DUID_BASE $PANDA_DIR/board/main.c
 
 # unused needs to run globally
-#printf "\n${GREEN}** UNUSED ALL CODE **${NC}\n"
-#cppcheck --enable=unusedFunction --quiet $PANDA_DIR/board/
+printf "\n${GREEN}** UNUSED ALL CODE **${NC}\n"
+cppcheck --enable=unusedFunction --quiet $PANDA_DIR/board/main.c 
```

Comment:
hmm, also not sure how this works now. it needs to also analyze the bootstub and jungle builds

Reply:
Seems like a bug from cppchecker's side. All the unused function seems to have been be generated by `board/main.c` as we can see in previous CI runs.
Running `cppcheck $PANDA_OPTS -DSTM32H7 -DUID_BASE $PANDA_DIR/board/main.c $PANDA_DIR/board/jungle/main.c $PANDA_DIR/board/jungle/bootstub.c` also seems to remove the unused function error ( it also adds another `unknownMacro` error )